codechefcli/problems.py

```diff
@@ -233,6 +237,63 @@ def search_problems(search_type):
         print(SERVER_DOWN_MSG)
 
 
+def get_tags(tags):
+    session = get_session()
+    if len(tags) == 0:
```

Comment:
It would be better to divide this function into 2 parts. One which will print tags and the other printing the problems.